server/src/v1/tests/user.test.js

```diff
@@ -86,7 +86,7 @@ describe.only('POST /api/v1/auth/signup', () => {
   });
 });
 
-describe('POST /api/v1/auth/signin', () => {
+describe.only('POST /api/v1/auth/signin', () => {
```

Comment:
Why only?

Reply:
I'm isolating that test suite, as im doing tdd I've tests already for the whole project and they are failing. I'm refactoring the tests one by one as making them pass. 
So the reason is to concentrate on the test, of the exact endpoint I'm currently working on.